src/main/java/com/google/javascript/gents/TypeConversionPass.java

```diff
@@ -309,16 +309,16 @@ public void visit(NodeTraversal t, Node n, Node parent) {
             if (!nodeAfterDefault.isName()) {
               continue;
             }
-            // It appears that adding ACCESS_MODIFIERs to Default params do not come out though
-            // the CodeGenerator, thus not safe to remove the declaration.
-            // TODO(rado): fix in emitting code and remove this line.
-            if (param.isDefaultValue()) {
-              continue;
-            }
             String paramName = nodeAfterDefault.getString();
             @Nullable
             JSTypeExpression paramType =
                 constructorJsDoc == null ? null : constructorJsDoc.getParameterType(paramName);
+            // The class member declaration on "this" will not have "=" in the JSDoc most of the time. Therefore we need to remove the "=" from the JSDoc of the parameter declaration so we can compare the types and collapse the parameter property.
```

Comment:
comment goes over 100+ chars

Reply:
Done.
`gradle googleJavaFormat` stopped working for me.

Reply:
Now I look at that, the code passed on travis so maybe our formatter setup is not working.